### PR TITLE
release v1.25.0

### PR DESCRIPTION
**Pull request by SaitejaTamma**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup

/Kind release

> /kind documentation
> /kind feature
> /kind hotfix

**What this PR does / Why we need it**: Release v1.25.0

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Guides/Client SDKs/unity.md [hunk 1]
```diff
@@ -29,27 +29,6 @@ Check the [Client SDK Documentation]({{< relref "_index.md" >}}) for more detail
 | Player Tracking      | PlayerConnect            | ❌️                            | 
 | Player Tracking      | PlayerDisconnect         | ❌️                            | 
 | Player Tracking      | SetPlayerCapacity        | ❌️                            | 
-{{% /feature %}}
-{{% feature publishVersion="1.25.0" %}}
```
Comment (markmandel) [inline site/content/en/docs/Guides/Client SDKs/unity.md:33]: You've removed the published version, not the expired version.

File: site/content/en/docs/Guides/Client SDKs/unity.md [hunk 2]
```diff
@@ -215,7 +194,6 @@ This returns a list of the playerIDs that are currently connected to the GameSer
 List<string> players = await agones.GetConnectedPlayers();
 ```
 
-{{% /feature %}}
```
Comment (markmandel) [inline site/content/en/docs/Guides/Client SDKs/unity.md:218]: This seems very odd, as you deleted the end of a feature code, but nothing from the beginning?

I figure this has broken lots of stuff in your PR. (Same in all the other ones as well)

**Comment by markmandel** [comment]
That's a flake I hadn't seen before:

```
	--feature-gates="" \
	--namespace=""
time="2022-08-03 18:26:55.512" level=info msg="Starting e2e test(s)" featureGates="CustomFasSyncInterval=true&Example=true&PlayerAllocationFilter=false&PlayerTracking=false&SDKGracefulTermination=false&StateAllocationFilter=false" gameServerImage="gcr.io/agones-images/simple-game-server:0.13" namespace= perfOutputDir= pullSecret= stressTestLevel=0 version=
time="2022-08-03 18:26:55.913" level=info msg="cause for namespace deletion error" reason=NotFound
time="2022-08-03 18:26:55.913" level=error msg="failed to cleanup e2e namespaces" error="namespaces \"1659550747\" not found"
FAIL	agones.dev/agones/test/e2e	0.499s
```

Restarting tests.

File: site/content/en/blog/releases/1.25.0.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- Move CustomFasSyncInterval to Beta [\#2646](https://github.com/googleforgames/agones/issues/2646)
+- Move NodeExternalDNS to stable [\#2643](https://github.com/googleforgames/agones/issues/2643)
+- Upgrade SuperTuxKart to 1.3 [\#2546](https://github.com/googleforgames/agones/issues/2546)
+- Docs: How to do local container with sdk [\#2677](https://github.com/googleforgames/agones/pull/2677) ([markmandel](https://github.com/markmandel))
+- upgrade xonotic version [\#2669](https://github.com/googleforgames/agones/pull/2669) ([mridulji](https://github.com/mridulji))
+- NodeExternalDNS/stable [\#2660](https://github.com/googleforgames/agones/pull/2660) ([SaitejaTamma](https://github.com/SaitejaTamma))
+- Move CustomFasSyncInterval to Beta [\#2654](https://github.com/googleforgames/agones/pull/2654) ([govargo](https://github.com/govargo))
+- Support for Unity Alpha SDK [\#2600](https://github.com/googleforgames/agones/pull/2600) ([MaxHayman](https://github.com/MaxHayman))
+
+Documentation: https://development.agones.dev/site/
```
Comment (markmandel) [inline site/content/en/blog/releases/1.25.0.md:31]:
```suggestion
```

This line should be dropped on full stable releases.

File: site/content/en/blog/releases/1.25.0.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+title: "1.25.0 - More enhancements and bug fixes"
```
Comment (markmandel) [inline site/content/en/blog/releases/1.25.0.md:2]:
```suggestion
title: "1.25.0 - Unity SDK Alpha Support, Fleet Custom Sync Interval and more"
```

Sound good?

File: site/content/en/docs/Guides/feature-stages.md [hunk 5]
```diff
@@ -24,7 +24,6 @@ that can be found in the [Helm configuration]({{< ref "/docs/Installation/Instal
 
 The current set of `alpha` and `beta` feature gates are:
 
-{{% feature expiryVersion="1.25.0" %}}
```
Comment (markmandel) [inline site/content/en/docs/Guides/feature-stages.md:27]: Shouldn't this entire block be deleted, not just the top and bottom feature codes?

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 6]
```diff
@@ -211,14 +211,12 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`                   | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition | `false`                |
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                             | `8000`                 |
 
-{{% feature publishVersion="1.25.0" %}}
```
Comment (markmandel) [inline site/content/en/docs/Installation/Install Agones/helm.md:214]: We should have a comment for this, this section is a special case.

For this section, take the configuration parameters from the table and move them up in the table above, and then increment the `publishVersion` in this featureCode block.

Basically we always keep a small "New Configuration Features" table ongoing between releases.

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 7]
```diff
@@ -294,7 +292,6 @@ For most use cases the controller would have required a restart anyway (eg: cont
 You can use our script located at {{< ghlink href="install/helm/agones/certs/cert.sh" >}}cert.sh{{< /ghlink >}} to generate them.
 {{< /alert >}}
 
-{{% feature publishVersion="1.22.0" %}}
```
Comment (markmandel) [inline site/content/en/docs/Installation/Install Agones/helm.md:297]: Oh that's an old one - nicely found!

File: site/content/en/docs/Reference/fleetautoscaler.md [hunk 8]
```diff
@@ -9,7 +9,6 @@ weight: 30
 A full `FleetAutoscaler` specification is available below and in the 
 {{< ghlink href="examples/fleetautoscaler.yaml" >}}example folder{{< /ghlink >}} for reference :
 
-{{% feature expiryVersion="1.25.0" %}}
```
Comment (markmandel) [inline site/content/en/docs/Reference/fleetautoscaler.md:12]: This whole block should be deleted.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,30 @@
 # Changelog
 
-## [v1.25.0-rc](https://github.com/googleforgames/agones/tree/v1.25.0-rc) (2022-07-26)
+## [v1.25.0](https://github.com/googleforgames/agones/tree/v1.25.0) (2022-08-02)
+
+[Full Changelog](https://github.com/googleforgames/agones/compare/v1.25.0-rc...v1.25.0)
+
+**Implemented enhancements:**
+
+- Support UDP Load Balancers to allow Agones to work with GKE Autopilot mode [\#2687](https://github.com/googleforgames/agones/issues/2687)
```
Comment (roberthbailey) [inline CHANGELOG.md:9]: This enhancement wasn't implemented and shouldn't be part of the changelog. I updated the label on the issue but to update this file you will need to regenerate the changelog.

Comment (markmandel) [inline CHANGELOG.md:9]:
```suggestion
```

**Review by markmandel — APPROVED** [review]

**Pull request merged**